From 837439354cd4692a7228f9f356e0c4acd32202f3 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 4 Feb 2020 20:11:16 -0500
Subject: [PATCH 1/1] clamav-unofficial-sigs.sh: fix malwarepatrol extended
 database name.

The database suffix needs to be ".db" regardless of whether you choose
the "basic" signatures or the "extended" ones. This patch is a quick
fix, and just forces the correct name at the beginning of the script.

Issue: https://github.com/extremeshok/clamav-unofficial-sigs/issues/300
---
 clamav-unofficial-sigs.sh | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/clamav-unofficial-sigs.sh b/clamav-unofficial-sigs.sh
index aa70db1..d8031c8 100644
--- a/clamav-unofficial-sigs.sh
+++ b/clamav-unofficial-sigs.sh
@@ -2235,11 +2235,7 @@ else
   fi
 fi
 
-if [ $malwarepatrol_list == "clamav_basic" ] ; then
-  malwarepatrol_db="malwarepatrol.db"
-else
-  malwarepatrol_db="malwarepatrol.ndb"
-fi
+malwarepatrol_db="malwarepatrol.db"
 malwarepatrol_url="${malwarepatrol_url}?receipt=${malwarepatrol_receipt_code}&product=${malwarepatrol_product_code}&list=${malwarepatrol_list}"
 
 # If "ham_dir" variable is set, then create initial whitelist files (skipped if first-time script run).
-- 
2.24.1

